dev/bots/codesign.dart

```diff
@@ -5,11 +5,10 @@
 import 'dart:io';
 import 'package:path/path.dart' as path;
 
-String get cacheDirectory {
-  final String flutterRepoRoot = path.normalize(path.join(path.dirname(Platform.script.path), '..', '..'));
-  return path.normalize(path.join(flutterRepoRoot, 'bin', 'cache'));
-}
+String get repoRoot => path.normalize(path.join(path.dirname(Platform.script.toFilePath()), '..', '..'));
```

Comment:
the previous of this script actually failed on cirrus because the space in the script file path was escaped. changing `Platform.script.path` -> `Platform.script.toFilePath()` fixed that.